tests/unit/semantic_release/hvcs/test_github.py

```diff
@@ -213,24 +213,16 @@ def test_pull_request_url(default_gh_client, pr_number):
     )
 
 
-def test_asset_upload_url(default_gh_client):
-    assert default_gh_client.asset_upload_url(
-        release_id=420
-    ) == "https://{domain}/repos/{owner}/{repo}/releases/{release_id}/assets".format(
-        domain=default_gh_client.hvcs_api_domain,
-        owner=default_gh_client.owner,
-        repo=default_gh_client.repo_name,
-        release_id=420,
-    )
-
-
 ############
 # Tests which need http response mocking
 ############
 
 
 github_matcher = re.compile(rf"^https://{Github.DEFAULT_DOMAIN}")
 github_api_matcher = re.compile(rf"^https://{Github.DEFAULT_API_DOMAIN}")
+github_upload_matcher = re.compile(
+    rf"^https://{Github.DEFAULT_API_DOMAIN.replace('api', 'uploads')}"
```

Comment:
I think this is cleaner if you add a constant to the `Github` class
```
DEFAULT_ASSET_UPLOAD_DOMAIN = DEFAULT_API_DOMAIN.replace("api", "uploads")
```
saves doing the replace everywhere